### PR TITLE
Fresh run of CIGALE for Heintz2020

### PR DESCRIPTION
**Pull request by profxj**
As titled.  New dust too.

### CODE REVIEW

File: frb/data/Galaxies/190102/FRB190102_host.json [hunk 1]
```diff
@@ -3,27 +3,28 @@
...
+        "AV_nebular": 0.09422530624456558,
+        "EBV_photom": 0.2749057999180598,
+        "EBV_photom_err": 0.17043260786874093,
+        "EBV_spec": 0.6095129619225207,
+        "Lnu_r": 3.8810223905041374e+21,
+        "Lnu_r_err": 2.159552516330269e+20,
+        "M_r": -19.872365370728126,
+        "M_r_err": 0.06041460515129164,
+        "Mstar": 4735151612.3161545,
+        "Mstar_err": 5442755343.36205,
```
Comment (keheintz) [inline frb/data/Galaxies/190102/FRB190102_host.json:15]: Error larger than value. Is this due to a conversion from log to linear scale?

File: frb/data/Galaxies/190102/FRB190102_host.json [hunk 2]
```diff
@@ -3,27 +3,28 @@
...
-        "EBV_photom": 0.10874369730986228,
-        "EBV_photom_err": 0.046784733194462205,
-        "EBV_spec": 0.6094787256151387,
-        "Lnu_r": 3.997132381967197e+21,
-        "Lnu_r_err": 2.6143450055200063e+20,
-        "M_r": -19.90437132999839,
-        "M_r_err": 0.07101313524233827,
-        "Mstar": 3178839293.9744,
-        "Mstar_err": 2129235298.7722566,
-        "Mstar_spec": 5104978902.866098,
```
Comment (keheintz) [inline frb/data/Galaxies/190102/FRB190102_host.json:16]: Error larger than value. Is this due to a conversion from log to linear scale?

**Review by keheintz — APPROVED** [review]

File: frb/data/Galaxies/190102/FRB190102_host.json (hunk 2 above)
Comment (profxj) [reply frb/data/Galaxies/190102/FRB190102_host.json:16]: There is no conversion.

**Comment by profxj** [comment]
merging

**Pull request merged**